interfaces/utils/utils.go

```diff
@@ -41,6 +45,14 @@ func NormalizeInterfaceAttributes(value interface{}) interface{} {
 			vc[key] = NormalizeInterfaceAttributes(item)
 		}
 		return vc
+	case json.Number:
```

Comment:
Can you please add a test for this case. I built the basics for it here: https://github.com/snapcore/snapd/pull/6012

Reply:
Added.